GameData/RealismOverhaul/Engine_Configs/Raptor.cfg

```
@@ -76,74 +78,125 @@
 		type = ModuleEngines
 		configuration = Raptor
 		modded = false
-		origMass = 1.6
+		origMass = 1.5
+
 		CONFIG
 		{
 			name = Raptor
-			maxThrust = 1779.29
-			minThrust = 355.85 // 20% throttle
-			PROPELLANT // Ratio = 3.8
+			maxThrust = 2241.90
+			minThrust = 896.76 // 40% throttle
+			PROPELLANT // Ratio = 3.55
 			{
 				name = LqdMethane
-				ratio = 0.4137
+				ratio = 0.4320
 				DrawGauge = true
 			}
 			PROPELLANT
 			{
 				name = LqdOxygen
-				ratio = 0.5863
+				ratio = 0.5680
 			}
 			atmosphereCurve
 			{
 				key = 0 355
 				key = 1 330
 			}
-			
-			ullage = True
-			pressureFed = False
-			ignitions = 5
-			
+
+			%ullage = True
+			%pressureFed = False
+			%ignitions = 0
+
 			IGNITOR_RESOURCE
 			{
 				name = ElectricCharge
 				amount = 0.5
 			}
-		
+			IGNITOR_RESOURCE // Touch-Lit System uses the same propellant has the engines
```

```suggestion
			IGNITOR_RESOURCE // Torch-Lit System uses the same propellant has the engines
```
Small typo, also present in other parts of the config. I won't highlight all of them, but there are 7 other occurrences